cloudlift/deployment/task_definition_creator.py

```diff
@@ -21,12 +21,13 @@ def _complete_image_url(ecr_client: EcrClient):
 
 
 class TaskDefinitionCreator:
-    def __init__(self, name, environment, version, build_args, region='ap-south-1'):
+    def __init__(self, name, environment, version, image_uri, build_args, region='ap-south-1'):
```

Comment:
Why `image_uri` positional argument in `TaskDefinitionCreator` constructor? I don't think deploy_service depend on `TaskDefinitionCreator`. Also the new positional argument might break `create_task_definition` & `update_task_definition`, could you please test & verify that?

Reply:
yes, it should not have been there. pushed the wrong changes.